include/crow/app.h

```diff
@@ -81,6 +84,12 @@ namespace crow
             return router_.new_rule_dynamic(std::move(rule));
         }
 
+//        ///Create a dynamic route for a blueprint using a rule (**Use CROW_BP_ROUTE instead**)
```

Comment:
Will be used later or something to delete?

Reply:
I forgot to delete this, my bad